apps/charterafrica/src/payload/blocks/Hero.js

```diff
@@ -0,0 +1,112 @@
+import link from "../fields/link";
+import richText from "../fields/richText";
+
+const Hero = {
+  slug: "hero",
+  fields: [
+    {
+      name: "title",
+      type: "group",
```

Comment:
I expect `hero` to be a block of `array` of `group`s. What am I missing?

Reply:
@kilemensi, From my view of the design, all pages have 1 `hero` section.

Reply:
@kelvinkipruto .

1. We're doing `hero` for the homepage.
2. It has multiple slides i.e. a carousel, and hence my comment above of `array` of `group`s with each slide being a group.
3. All other pages with header/heroes use a different structure from that used on homepage.